talk/expert/modules.tex

```diff
@@ -0,0 +1,510 @@
+\subsection[modules]{Modules}
+
+\begin{frame}[fragile]
+  \frametitlecpp[20]{Modules}
+  \begin{block}{Motivation}
+    \begin{itemize}
+    \item Better isolation
+    \begin{itemize}
+      \item Between multiple pieces of included code
+      \item Between included and your code
+    \end{itemize}
+    \item Better control over public interface of your library
+    \begin{itemize}
+      \item Avoid leaking library dependencies into user code
+    \end{itemize}
+    \item Faster compilation
+    \end{itemize}
```

Comment:
As the very first thing, I would have liked to see what a module is. Here's a try:
```suggestion
    \begin{itemize}
      \item Modules allow for sharing declarations and definitions across translation units
      \item Header files do the same, but modules have benefits:
      \begin{itemize}
        \item Better isolation. No cross-talk between multiple pieces of included code, and also not between included code an your code.
        \item Better control over public interface of your library. Avoid leaking library dependencies into user code.
        \item Faster compilation
      \end{itemize}
    \end{itemize}
```

Reply:
Not bad, I take that.